pkg/config/agent/config.go

```diff
@@ -102,14 +102,21 @@ type AgentConfig struct {
 	TrafficEncryptionMode string `yaml:"trafficEncryptionMode,omitempty"`
 	// WireGuard related configurations.
 	WireGuard WireGuardConfig `yaml:"wireGuard"`
+	// Eanble bridging mode of Pod network on Nodes, in which the Node's transport interface is connected
+	// to the OVS bridge, and cross-Node/VLAN traffic from AntreaIPAM Pods (Pods whose IP addresses are
+	// allocated by AntreaIPAM from IPPools) is sent to the underlay network via the uplink, and
+	// forwarded/routed by the underlay network.
+	// This option requires the `AntreaIPAM` feature gate to be enabled. At this moment, it supports only
```

Comment:
I'm not sure I understand this part. EnableBridgingMode requires AntreaIPAM or AntreaIPAM requires EnableBridgingMode? I think technically EnableBridgingMode can just work without AntreaIPAM enabled, but just doesn't make any difference from function's perspective?

Reply:
The current situation is:

- AntreaIPAM is a feature gate for both AntreaIPAM and bridging mode (connecting uplink to bridge, flexible IPAM for Pods, and datapath changes for such Pods).
- EnableBridgingMode is a config option to enable bridging mode.